specutils/tests/test_analysis.py

```diff
@@ -460,6 +460,23 @@ def test_fwhm():
     assert quantity_allclose(result, expected, atol=0.01*u.GHz)
 
 
+    # Highest point at the first point
+    frequencies = np.linspace(1, 10, 100) * u.um
```

Comment:
(minor) this isn't actually a frequency - it's a wavelength, so the variable name is a bit confusing